packages/L10n/src/locales/ja.yml

```diff
@@ -4,6 +4,8 @@ ja:
     remove: 削除
     moreInformation: 詳しい情報
     close: 閉じる
+    dataGrid:
+      a11yTextMessage: "{{value}}. 行 {{rowIndex}}、列 {columnIndex}} の位置にいます。次の情報は無視してください。"
```

Comment:
Missing opening bracket in: {columnIndex}}

Reply:
Good catch!

Reply:
@cristinajimenez fixed. needs an 🆗 now